redisdb/tests/test_e2e.py

```diff
@@ -75,6 +75,7 @@ def assert_common_metrics(aggregator):
     aggregator.assert_metric('redis.key.length', count=2, tags=(['key:test_key1', 'key_type:list'] + tags_with_db))
     aggregator.assert_metric('redis.key.length', count=2, tags=(['key:test_key2', 'key_type:list'] + tags_with_db))
     aggregator.assert_metric('redis.key.length', count=2, tags=(['key:test_key3', 'key_type:list'] + tags_with_db))
+    aggregator.assert_metric('redis.key.length', count=2, tags=(['key:test_key4', 'key_type:stream'] + tags_with_db))
```

Comment:
I'm not sure that's enough to make sure the bug you saw is fixed. I think what we should do is: 

```suggestion
    aggregator.assert_metric('redis.key.length', value=2, count=2, tags=(['key:test_key4', 'key_type:stream'] + tags_with_db))
```

and `master.xadd` the `test_key4` once more (see my other comment). WDYT?